config/application.rb

```diff
@@ -9,11 +9,14 @@
 module MovieApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults 7.0
```

Comment:
This was increased after updating all the defaults in the generated file and verifying the tests pass